### PR TITLE
Clean up dockerfile 

### LINKED ISSUE #15: Use upper case for Dockerfile commands
Github supports syntax highlighting only if the docker file commands are in upper case. eg: https://github.com/9at8/room-leds/blob/master/Dockerfile

### LINKED ISSUE #16: Clean apt-get files
After apt-get update and apt-get install run `apt-get clean && rm -rf /var/lib/apt/lists/*` to reduce size of the docker image.

### PR DESCRIPTION
**Pull request by Azhng**
Close #16 
Close #15 

cc @arora-aman

### CODE REVIEW

**Review by arora-aman — APPROVED** [review]
LGTM

**Comment by Azhng** [comment]
Will merge once #11  is merged

**Pull request merged**
